pom.xml

```diff
@@ -49,11 +49,9 @@
         <module>EssentialsProtect</module>
         <module>EssentialsSpawn</module>
         <module>EssentialsXMPP</module>
-        <module>nms/NMSProvider</module>
-        <module>nms/UpdatedMetaProvider</module>
-        <module>nms/LegacyProvider</module>
-        <module>nms/ReflectionProvider</module>
-        <module>nms/FlattenedProvider</module>
+        <module>providers/BaseProviders</module>
+        <module>providers/NMSReflectionProvider</module>
+        <module>providers/PaperProvider</module>
```

Comment:
Name suggestions: "Base", "Reflection", "Paper", to be less redundant, since this is all already in providers/

Reply:
The module names are named as such because out of context it could be confusing

Reply:
I would still suggest at least NMSReflectionProvider -> ReflectionProvider, because not all reflection will be NMS (which is the case with some code in the other open PR).